universal-login-relayer/lib/config/relayer.ts

```diff
@@ -30,7 +31,11 @@ const config: Config =  Object.freeze({
     getEnv('ENS_DOMAIN_2', ''),
     getEnv('ENS_DOMAIN_3', ''),
   ],
-  walletMasterAddress: getEnv('WALLET_MASTER_ADDRESS', '')
+  walletMasterAddress: getEnv('WALLET_MASTER_ADDRESS', ''),
+  contractWhiteList: {
+    master: [],
+    proxy: []
```

Comment:
Please state directly supported list of contracts (hash)

Reply:
✔️